resources/js/svir_viewer_d3_tree.js

```diff
@@ -86,26 +89,34 @@
         function createSpinner(id, weight, name, field) {
             pdTempSpinnerIds.push("spinner-"+id);
             $('#projectDefWeightDialog').dialog("open");
-            if (field === undefined) {
-                $('#projectDefWeightDialog').append(
-                    '<p><label for="spinner'+id+'">'+name+': </label><input id="spinner-'+id+'" name="spinner" value="'+weight+'"></p>');
-            } else {
-                $('#projectDefWeightDialog').append(
-                    '<p><label for="spinner'+id+'">'+name+'('+field+'): </label><input id="spinner-'+id+'" name="spinner" value="'+weight+'"></p>');
+            var content = '<p><label style="clear: left; float: left; width: 10em;" for="spinner'+id+'">'+name;
+            if (typeof field !== 'undefined') {
+                content += ' ('+field+')';
             }
+            content += ': </label><input id="spinner-'+id+'" name="spinner" value="'+Math.abs(weight)+'">';
+            content += '<input type="checkbox" id="inverter-spinner-'+id+'"><label style="font-size: 0.8em; "for="inverter-spinner-'+id+'" title="Select to invert the contribution of the variable to the calculation">Invert</label>';
```

Comment:
same as above with regards to the styling. 
Also consider line breaks like this:
'<input type="checkbox" id="inverter-spinner-'+id+'">' + 
'<label style="font-size: 0.8em; "for="inverter-spinner-'+id+'"' + 
'title="Select to invert the contribution of the variable to the calculation">Invert</label>';